packages/react/src/components/DataTable/DataTable.types.ts

```diff
@@ -196,6 +197,12 @@ export interface DataTableProps<TData extends RowData> {
       index: number,
       keyword: string,
     ) => number | string;
+    /**
+     * Whether the table header should stick when scrolling vertically
```

Comment:
I am wondered why do we have to copy-paste this.
Can we do `footerProps?: FooterPropsType;` ?

Reply:
cuz i wanted the props to look like these in storybook controls
![image](https://github.com/project44/manifest/assets/133108657/9acefbfa-5f31-4ad4-9f4a-6f981a2924f7)

Reply:
Not sure this is so important that duplication of code is necessary

Reply:
okay then ill remove it